packages/dht/src/connection/WebSocket/WebSocketConnector.ts

```diff
@@ -263,7 +299,8 @@ export class WebSocketConnector implements IWebSocketConnectorService {
     public async destroy(): Promise<void> {
         this.destroyed = true
         this.rpcCommunicator.stop()
-
+        this.autocertifierRpcCommunicator.stop()
+        this.autoCertifierClient?.stop()
```

Comment:
Let's choose one naming convention for capitalization. (Or just remove the "auto" word to make things simpler)